perl-package/AI-MXNet/lib/AI/MXNet/Metric.pm

```diff
@@ -20,6 +20,7 @@ use strict;
 use warnings;
 use AI::MXNet::Function::Parameters;
 use Scalar::Util qw/blessed/;
+use JSON::PP;
```

Comment:
I think normally people just use `JSON` and that module makes a good choice about using `JSON::PP` or `JSON::XS`.

Reply:
JSON::PP is in perl core, JSON or JSON::XS are not, this had motivated my decision.

Reply:
ah, news to me. thanks!